idx/idx_gen.go

```diff
@@ -1,70 +1,249 @@
 package idx
 
-// Code generated by github.com/tinylib/msgp DO NOT EDIT.
+// NOTE: THIS FILE WAS PRODUCED BY THE
```

Comment:
Looks like this file was generated with an old version of msgp.

see https://github.com/grafana/metrictank/blob/master/scripts/qa/go-generate.sh

Reply:
sorry about that, it's fixed now